@here/harp-mapview/lib/ThemeLoader.ts

```diff
@@ -50,6 +50,20 @@ export interface ThemeLoadOptions {
      */
     resolveDefinitions?: boolean;
 
+    /**
+     * Resolve the URIs to resources like fonts, icons, ...
+     * If true, [[uriResolver]] will be used to resolve the URI
+     * @default true
+     */
+    resolveResourceURIs?: boolean;
```

Comment:
Minor, i think that proper camel case would be `resolveResourceUris` as, acronyms should be written as they are normal, lower case words. At least most (`Omv*`) of our code base follows this convention https://stackoverflow.com/questions/15526107/acronyms-in-camelcase

Reply:
done